### PR TITLE
Add foot

### PR DESCRIPTION
**Pull request by p00f**

### CODE REVIEW

**Comment by lunaryorn** [comment]
Thanks. Please add it to README and CHANGELOG.

**Comment by p00f** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 To publish a new release run `scripts/release` from the project directory.
 
 ## [Unreleased]
-
+- Support for [foot](https://codeberg.org/dnkl/foot/) (see [GH-190])
```
Comment (lunaryorn) [inline CHANGELOG.md:10]: Please follow the changelog format here, by adding a corresponding section header.

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ To publish a new release run `scripts/release` from the project directory.
 [GH-182]: https://github.com/lunaryorn/mdcat/pull/182
 [GH-183]: https://github.com/lunaryorn/mdcat/issues/183
 [GH-184]: https://github.com/lunaryorn/mdcat/pull/184
+[GH-190]: https://github.com/lunaryorn/mdcat/pull/190
```
Comment (lunaryorn) [inline CHANGELOG.md:34]: Please remove this reference definition from the 0.23.3 section to the unreleased section.

**Review by lunaryorn — CHANGES_REQUESTED** [review]

**Comment by p00f** [comment]
Done

**Review by lunaryorn — APPROVED** [review]
Thanks a lot 🙏

**Pull request merged**